documentation/pysa_github_action/README.md

```diff
@@ -0,0 +1,69 @@
+# Pysa Github Action
+
+Run pysa on the repository in Github Actions and create a SARIF File.
+
+## Dependencies
+
+The action requires a python setup with the python version > 3.5
```

Comment:
Is there a way to specify we want a python version > 3.7? SAPP won't run on python versions earlier than 3.7, because of some type annotations we use

Reply:
The composite action makes use of the existing version of python the action uses. So the thing we can do is, make a comment. We can maybe, migrate this action to a node js action if we require total control.

Reply:
Can we use the following here?
```
    strategy:
      matrix:
        python-version: [3.6, 3.7, 3.8, 3.9]
```

https://docs.github.com/en/actions/guides/building-and-testing-python#starting-with-the-python-workflow-template

Reply:
Unfortunately that is for a GitHub action -> not something that we can use here (because composite actions don't support uses statement yet). However, I will add this as a comment.